src/Shopsys/ShopBundle/DataFixtures/resources/demo-data-products.csv

```diff
@@ -1,148 +1,148 @@
-JURA Impressa J9 TFT Carbon;JURA Impressa J9 TFT Carbon;9772572;JURA Impressa J9 TFT Carbon;8845781245933;JURA Impressa J9 TFT Carbon Automatick� k�vovar/espresso,barevn� LCD;"<h2>JURA IMPRESSA Z9 One Touch TFT Pianoblack</h2><p> Luxury automatic coffee machine Jura Impressa Z9 One Touch packed with Swiss precision and elegance, which sets new standards. This machine brings you jeich on several levels, such as coffee quality, easy to use and attractive design. The purpose is always the most advanced technology to achieve real, authentic results. The powerful device has intuitive controls and except large 3.5 ""TFT color display. Together with rotary dial allows you to choose between 11 specialties and goes to prepare any coffee you can think of. With the coffee a la carte, allowing you to quickly change settings according to your needs, your options are almost limitless. </p><p> For an authentic experience and to ensure optimal conditions for coffee are two Thermoblocks. The first coffee and hot water, the second is intended to couple. With every cup of coffee is also possible to set the correct brewing temperature, amount of water and grind. Mill technology + Aroma is quiet and up to twice
```

Comment:
Empty short description and description should remain empty in German translation as well imo